src/applications/personalization/dashboard/containers/AppointmentsWidget.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+import { startCase } from 'lodash';
+
+import {
+  fetchAppointments
+} from '../../appointments/actions';
+
+import LoadingIndicator from '@department-of-veterans-affairs/formation/LoadingIndicator';
+
+class AppointmentsWidget extends React.Component {
+  componentDidMount() {
+    if (!this.props.loading) {
+      this.props.fetchAppointments();
+    }
+  }
+
+  render() {
+    if (this.props.loading) {
+      return <LoadingIndicator message="Loading your appointments..."/>;
+    }
+
+    return (
+      <div className="appointments-container">
+        <h3>Appointments</h3>
+        {this.props.appointments.data.map((a, ix) => {
+          return (
```

Comment:
It might be useful to define the shape of the Appointment array as a Proptype, to help document what properties are available for later without having to inspect a network payload.